src/ESLintLinter.php

```diff
@@ -45,6 +46,21 @@ public function getLinterConfigurationName() {
   }
 
   public function getDefaultBinary() {
+    if ($this->cwd) {
+      $realCWD = Filesystem::resolvePath($this->cwd, $this->getProjectRoot());
+      list($err, $stdout, $stderr) = exec_manual('yarn -s --cwd %s bin eslint', $realCWD);
+      if ($stdout) {
+        return strtok($stdout, "\n");
+      }
+    } else {
+      $localBinaryPath = Filesystem::resolvePath('./node_modules/.bin/eslint', true);
```

Comment:
The second argument to `resolvePath` should be the relative directory. I think you don't need to pass a second argument here at all.

```php
  /**
   * Canonicalize a path by resolving it relative to some directory (by
   * default PWD), following parent symlinks and removing artifacts. If the
   * path is itself a symlink it is left unresolved.
   *
   * @param  string    Path, absolute or relative to PWD.
   * @return string    Canonical, absolute path.
   *
   * @task   path
   */
  public static function resolvePath($path, $relative_to = null) {
```